docs/index.js

```diff
@@ -695,6 +711,10 @@ class Task {
     this.isEditing = false;
   }
 
+  toggleStatus = () => {
+    this.completed = !this.completed;
+  }
+
   createNode = () => {
```

Comment:
[Optional] Please consider clearing the input field after a task has been created. This will greatly improve the user experience as they don't need to manually clear the field for each new task